app/components/TheSearchBar.vue

```diff
@@ -0,0 +1,58 @@
+<script setup lang="ts">
+import type { PropType } from 'vue'
+import type { SelectValue } from 'ant-design-vue/es/select'
+import type { Mlm } from '../types'
```

Comment:
Can we import this using 

```suggestion
import type { Mlm } from '~/types'
```